SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -99,9 +117,13 @@ public override void OnChannelGroupReady(IChannelGroup channelGroup, string appS
             {
                 base.OnChannelGroupReady(channelGroup, appSecret);
                 ApplyEnabledState(InstanceEnabled);
+                if (InstanceEnabled)
+                {
+                    ProcessPendingErrors();
```

Comment:
This method does a lot of file IO. Is this running on the proper thread?

Reply:
Isn't it already in a background thread? My understand is that `OnChannelGroupReady` is called in a background thread already.

Reply:
Hmm. Looks it isn't running in a background thread. I'll change that.